src/main/java/kitchenpos/order/domain/OrderLineItem.java

```diff
@@ -0,0 +1,89 @@
+package kitchenpos.order.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import kitchenpos.common.Quantity;
+import kitchenpos.menu.domain.Menu;
+import kitchenpos.menu.exception.NotFoundMenuException;
+import kitchenpos.order.exception.NotFoundOrderException;
+
+@Entity
+@Table(name = "order_line_item")
+public class OrderLineItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "seq")
+    private Long seq;
```

Comment:
어플리케이션 내부에서는 id라는 이름으로 가져가고 데이터베이스 컬럼 이름을 seq로 사용하는건 어떨까요 ? :)

데이터베이스의 상태와 별도로 어플리케이션의 변수명은 일관성을 가져가야한다고 생각합니다 🤔

```suggestion
    @Column(name = "seq")
    private Long id;
```

Reply:
넵 `id`로 통일해둘게요~